### DISCUSSION TITLE
null-ls is archived... what other linter/formatter will be used

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [LunarVim/LunarVim#4571] [General]
So, [null-ls](https://github.com/jose-elias-alvarez/null-ls.nvim/issues/1621) is archived... 
When it will be removed from Lunar and what is the next?
Should I bother?

**Comment by juicy-g** [comment]
LunarVim has already been switched to [none-ls](https://github.com/nvimtools/none-ls.nvim) to replace null-ls in the latest release. none-ls is a community maintained replacement to null-ls. There should be no action on your part other than upgrade to LunarVim 1.4.
